src/components/PassphraseExample.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import cx from 'classnames'
+import styles from 'styles/passphrase'
+
+const alphaRegexp = /^[a-z]$/i
+const numberRegexp = /^[0-9]$/
+
+const isAlphaCharacter = char => alphaRegexp.test(char)
+const isNumberCharacter = char => numberRegexp.test(char)
+
+const tokenTypes = {
+  alpha: 'alpha',
+  number: 'number',
+  special: 'special'
+}
+
+const getTokenType = char => {
+  if (isAlphaCharacter(char)) {
+    return tokenTypes.alpha
+  }
+
+  if (isNumberCharacter(char)) {
+    return tokenTypes.number
+  }
+
+  return tokenTypes.special
+}
+
+const tokenize = passphrase => {
+  const tokens = passphrase.split('').map(char => ({
+    char,
+    type: getTokenType(char)
+  }))
+
+  return tokens
+}
```

Comment:
I don't understand the manipulation we're doing on the passphrase. Can you explain why we have to map on each char to check the Type?

Reply:
Oh, is it to check the strength of the password?

Reply:
See https://github.com/cozy/cozy-settings/pull/269#discussion_r357227962